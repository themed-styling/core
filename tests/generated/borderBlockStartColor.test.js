import 'jest-styled-components'

import { borderBlockStartColor } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(borderBlockStartColor, 'borderBlockStartColor')
