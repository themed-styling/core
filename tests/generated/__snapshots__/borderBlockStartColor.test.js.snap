// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`borderBlockStartColor renders 0 1`] = `
Array [
  .c0 {
  border-block-start-color: #0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #0;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #0;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders function 1`] = `
Array [
  .c0 {
  border-block-start-color: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: has;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: has;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders invalid theme path 1`] = `
Array [
  .c0 {
  border-block-start-color: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders negative number 1`] = `
Array [
  .c0 {
  border-block-start-color: #-54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #-54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #-42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #-42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  border-block-start-color: July;
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: July;
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    border-block-start-color: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    border-block-start-color: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-block-start-color: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-block-start-color: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  border-block-start-color: July;
}

@media May {
  .c0 {
    border-block-start-color: August;
  }
}

@media April {
  .c0 {
    border-block-start-color: September;
  }
}

@media January {
  .c0 {
    border-block-start-color: November;
  }
}

@media April {
  .c0 {
    border-block-start-color: August;
  }
}

@media June {
  .c0 {
    border-block-start-color: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: July;
}

@media May {
  .c0 {
    border-block-start-color: August;
  }
}

@media April {
  .c0 {
    border-block-start-color: September;
  }
}

@media January {
  .c0 {
    border-block-start-color: November;
  }
}

@media April {
  .c0 {
    border-block-start-color: August;
  }
}

@media June {
  .c0 {
    border-block-start-color: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-color: November;
  }
}

@media April {
  .c0 {
    border-block-start-color: January;
  }
}

@media January {
  .c0 {
    border-block-start-color: December;
  }
}

@media April {
  .c0 {
    border-block-start-color: April;
  }
}

@media June {
  .c0 {
    border-block-start-color: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-color: November;
  }
}

@media April {
  .c0 {
    border-block-start-color: January;
  }
}

@media January {
  .c0 {
    border-block-start-color: December;
  }
}

@media April {
  .c0 {
    border-block-start-color: April;
  }
}

@media June {
  .c0 {
    border-block-start-color: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    border-block-start-color: July;
  }
}

@media April {
  .c0 {
    border-block-start-color: August;
  }
}

@media January {
  .c0 {
    border-block-start-color: September;
  }
}

@media April {
  .c0 {
    border-block-start-color: November;
  }
}

@media June {
  .c0 {
    border-block-start-color: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    border-block-start-color: July;
  }
}

@media April {
  .c0 {
    border-block-start-color: August;
  }
}

@media January {
  .c0 {
    border-block-start-color: September;
  }
}

@media April {
  .c0 {
    border-block-start-color: November;
  }
}

@media June {
  .c0 {
    border-block-start-color: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-color: January;
  }
}

@media April {
  .c0 {
    border-block-start-color: December;
  }
}

@media January {
  .c0 {
    border-block-start-color: April;
  }
}

@media April {
  .c0 {
    border-block-start-color: May;
  }
}

@media June {
  .c0 {
    border-block-start-color: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-block-start-color: January;
  }
}

@media April {
  .c0 {
    border-block-start-color: December;
  }
}

@media January {
  .c0 {
    border-block-start-color: April;
  }
}

@media April {
  .c0 {
    border-block-start-color: May;
  }
}

@media June {
  .c0 {
    border-block-start-color: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders postitive number 1`] = `
Array [
  .c0 {
  border-block-start-color: #54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: #42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders string 1`] = `
Array [
  .c0 {
  border-block-start-color: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: June;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: June;
}

<div
    className="c0"
  />,
]
`;

exports[`borderBlockStartColor renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`borderBlockStartColor renders valid theme path 1`] = `
Array [
  .c0 {
  border-block-start-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-block-start-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
